spec/linter-eslint-spec.js

```diff
@@ -128,21 +140,70 @@ describe('The eslint provider for Linter', () => {
     })
   })
 
-  describe('Fix errors when saved', () => {
+  describe('Fix errors', () => {
+    let editor
+    let doneCheckingFixes
+    let tempFixtureDir
+    let tempFixturePath
+    let tempConfigPath
+
     beforeEach(() => {
-      atom.config.set('linter-eslint.fixOnSave', true)
+      waitsForPromise(() => {
+        tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
+        tempFixturePath = `${tempFixtureDir}${path.sep}fixed.js`
```

Comment:
Hah, yeah power of suggestion [(from the node docs)](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback).  `path.join` would indeed be better, I'll switch it out.